README.md

```diff
@@ -1,3 +1,6 @@
+# Live Demo
+https://akeegandev.github.io/math_magicians/
+
 # Getting Started with Create React App
```

Comment:
The Readme is not following the correct template. Please check out https://github.com/microverseinc/readme-template for inspiration.